### DISCUSSION TITLE
Ability to dynamically set default placeholder

### DISCUSSION THREAD

**Discussion by Braunson** [wire-elements/spotlight#56] [Ideas]
It would be great to be able to dynamically set the placeholder to morning/afternoon/evening or even based on user-type OR to include the user's name. If there's interest, I can put in a PR.

**Comment by PhiloNL** [comment]
Hi @Braunson,
Feel free to create a PR if this is something you need 😄

**Comment by bebo925** [comment]
I created #59 to allow for passing in a parameter to override the the default placeholder.  It should be backwards compatible and allow you to do want to do.  I thought through a few ways to handle it, but I ultimately found a way to do it by changing only 2 lines of code.

**Reply by Braunson** [reply]
@bebo925 Might I suggest changing `$placeholderOverride` to just `$placeholder`, I think it's a decent solution though.

**Reply by bebo925** [reply]
I agree and went with that first, but I talked myself out of it.  I updated the PR accordingly.
